cmd/db.go

```diff
@@ -22,6 +22,7 @@ type database interface {
 	CreateCompanies([][]string) error
 	PostLoad() error
 	MetaSave(string, string) error
+	ExtraIndexes(idxs []string) error
```

Comment:
Esse comando não pertence nem ao `transform` nem à `api`. Precisamos de um novo bloco para ele.

```suggestion
        // extra indexes
	ExtraIndexes(idxs []string) error
```